pkg/apiserver/apiserver.go

```diff
@@ -78,6 +78,10 @@ type Mux interface {
 type APIGroupVersion struct {
 	Storage map[string]rest.Storage
 
+	// Root is the APIPrefix under which this is being served.  It can also be the APIPrefix/APIGroup that is being served
```

Comment:
> May we merge #14156 first (it's quite big and will be painful to rebase), and then split the fields in another PR?

As long as we can agree to do it and I'd rather not make it a requirement to change every instance in every type at the same time.